Packs/AlibabaActionTrail/ReleaseNotes/1_0_12.md

```diff
@@ -0,0 +1,3 @@
+
+#### Triggers Recommendations
+- Updated the trigger recommendations.
```

Comment:
```suggestion
#### Triggers Recommendations
##### Alibaba ActionTrail - Multiple Unauthorized Action Attempts Detected By a User Alerts
- Updated the trigger recommendations.
```

Reply:
@altmannyarden asked not to include the trigger name in the RN.
Since the files and YAML validation failed, I added the trigger name and excluded it.